src/components/RouteSelector.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <v-menu>
```

Comment:
Can you please use the [`v-navigation-drawer`](https://vuetifyjs.com/en/components/navigation-drawers/) here instead?

Reply:
@GrumpyBear57 I can try to switch it over.